faw/ci/ci_container_service.py

```diff
@@ -753,7 +751,10 @@ def _create_dockerfile_contents(development, config, config_data, build_dir, bui
         # Development extensions... add not-compiled code directories.
         dockerfile_final.append(r'''
             # Install npm globally, so it's available for debug mode
-            RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - && apt-get install -y nodejs
+            RUN wget -qO - https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | apt-key add -
+            RUN echo "deb https://deb.nodesource.com/node_16.x nodistro main" > /etc/apt/sources.list.d/nodesource.list
+            RUN apt-get update
```

Comment:
Always put `apt-get update` on same line as `apt-get -y install`, otherwise docker cache can be confusing when the install line is changed

Reply:
e.g., `apt-get update && apt-get -y install ...`